tests/app/celery/test_contact_information_tasks.py

```diff
@@ -314,3 +319,68 @@ def test_exception_sets_failure_reason_if_thrown(
             )
 
     mocked_check_and_queue_callback_task.assert_called_once_with(notification)
+
+
+@pytest.mark.parametrize(
+    'default_send, user_set, expected',
+    [
+        # If the user has set a preference, we always go with that and override default_send
+        [True, True, True],
+        [True, False, False],
+        [False, True, True],
+        [False, False, False],
+        # If the user has not set a preference, go with the default_send
+        [True, None, True],
+        [False, None, False],
+    ],
+)
+@pytest.mark.parametrize('notification_type', [CommunicationChannel.EMAIL, CommunicationChannel.TEXT])
+def test_ut_get_email_or_sms_with_permission_utilizes_default_send(
```

Comment:
There's no problem with this, but we discussed that we're not distinguishing between "ut", etc., anymore.

Reply:
Sure, I can delete those. Was copy/paste.